apps/nextcloud-fpm/configure-scripts/configure-scripts/occ-notify-push.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+occ_notify_push_install() {
+  echo '## Configuring Notify Push...'
+  install_app notify_push
```

Comment:
```suggestion
  install_app notify_push || echo "installing Notify_Push failed, please check internet connectivity..."
```

we need to show a clear error message, theoretically his might fail on firewalled/proxied/airgapped installs, which are not adviced, unlikely etc.

Reply:
Lets do this on the `install_app` function instead once. 
As it might also not return the correct exit code to catch that here